llvm/utils/TableGen/RegisterBankEmitter.cpp

```diff
@@ -289,6 +299,141 @@ void RegisterBankEmitter::emitBaseClassImplementation(
      << "} // end namespace llvm\n";
 }
 
+// TableGen already had some RegisterBankInfo support:
```

Comment:
Don't need the history lesson

Reply:
Yeah, that was mostly for me. I'll simplify it.